### PR TITLE
Do not use root user for puppetca

### PR DESCRIPTION
**Pull request by elmatze79**

### CODE REVIEW

File: igvm/vm.py [hunk 1]
```diff
@@ -455,11 +455,10 @@ def run_puppet(self, clear_cert=False, debug=False):
         if clear_cert:
             with settings(
                 host_string=self.dataset_obj['puppet_ca'],
-                user='root',
                 warn_only=True,
             ):
                 run(
-                    '/usr/bin/puppet cert clean {}'.format(self.fqdn),
+                    'sudo /usr/bin/puppet cert clean {}'.format(self.fqdn),
```
Comment (TuxPowered42) [inline igvm/vm.py]: Please never directly type `sudo` in commands, we have a parameter of `run()` for that.

File: igvm/vm.py [hunk 2]
```diff
@@ -455,10 +455,9 @@ def run_puppet(self, clear_cert=False, debug=False):
         if clear_cert:
             with settings(
                 host_string=self.dataset_obj['puppet_ca'],
-                user='root',
                 warn_only=True,
             ):
-                run(
+                sudo(
```
[the remark this answers is not in the document]
Comment (elmatze79) [reply igvm/vm.py]: E       TypeError: run() got an unexpected keyword argument 'with_sudo'

[the remark this answers is not in the document]
Comment (TuxPowered42) [reply igvm/vm.py]: Now I see the issue. This is run *directly* against puppet server via Fabric, not through our normal `Host` object. In this case Fabric API call `sudo` seems the best option but you must import it first.

**Comment by Woellchen** [comment]
fixed with #130

**Pull request closed without merging**